Spanner/src/Operation.php

```diff
@@ -210,7 +216,7 @@ public function rollback(Session $session, $transactionId, array $options = [])
      * }
      * @return Result
      */
-    public function execute(Session $session, $sql, array $options = [])
+    public function execute(Session $session, $sql, array $options = [], &$transactionId = null)
```

Comment:
We do not want to pass by reference. There are many reasons why this is bad practice that we can discuss, but we need to find an approach where we aren't doing this.

Also, since this is optional, why not make `transactionId` a value in `$options`?